docs/HyperIndex/Guides/configuration-file.mdx

```diff
@@ -77,20 +70,10 @@ More information on Human Readable ABI parsing is available [here](https://docs.
 - Address field should contain the address of the proxy contract, which emits the events on the specified blockchain.
 - Should the human readable ABI format not be used, then the ABI which is referenced in config file needs to be copied from the implementation contract into the specified abi directory.
 
-## Example `config.yaml` from Greeter template using ReScript language:
 
-```yaml
-name: Greeter
-description: Greeter indexer
-networks:
-  - id: 137 # Polygon
-    start_block: 45336336
-    contracts:
-      - name: PolygonGreeter
-        abi_file_path: abis/greeter-abi.json
-        address: 0x9D02A17dE4E68545d3a58D3a20BbBE0399E05c9c
-        handler: ./src/EventHandlers.bs.js
-        events:
-          - event: NewGreeting
-          - event: ClearGreeting
-```
+import CodeBlock from '@theme/CodeBlock';
+import Schema from "@site/static/schemas/config.evm.json";
+import JSONSchemaViewer from "@theme/JSONSchemaViewer";
+
+## Config Schema Explorer
```

Comment:
```suggestion
## Explorer
```

I suggest to simply call it `Explorer`, since the `Config Schema` is already mentioned in the explorer below


<img width="542" alt="image" src="https://github.com/user-attachments/assets/491cb471-c08d-4159-87b6-313d00b149fe">

Reply:
Sure done, I had to update the back link as well so didn't commit the suggestion 👍🏼